packages/builder/src/components/design/settings/controls/FieldConfiguration/FieldConfiguration.svelte

```diff
@@ -145,16 +148,35 @@
     dispatch("change", getValidColumns(parentFieldsUpdated, options))
   }
 
-  const listUpdated = e => {
-    const parsedColumns = getValidColumns(e.detail, options)
+  const listUpdated = columns => {
+    const parsedColumns = getValidColumns(columns, options)
     dispatch("change", parsedColumns)
   }
+
+  const toggleAll = update => {
+    listUpdated(update)
+  }
 </script>
 
 <div class="field-configuration">
+  <div class="toggle-all">
+    <span />
+    <Toggle
+      on:change={() => {
+        let update = fieldList.map(field => ({
+          ...field,
+          active: selectAll,
+        }))
+        toggleAll(update)
```

Comment:
We should either remove the `toggleAll` function now (since its a straight pass-through to `listUpdate`) or better yet in my opinion, move the inline logic here into toggle all and keep it.